modules/references/client/less/recommend.less

```diff
@@ -0,0 +1,3 @@
+.recommend-reply-button {
+  margin: 0 1em 0 1em !important;
```

Comment:
Let's avoid `important` and ass CSS specificity if there's a problem applying this otherwise. So e.g. add `.btn.recommend-reply-button`.

`0 1em` can be applied together to shorthand (two values Vs four).

Reply:
Yeah, I was also annoyed by the `!important` statement.
Thank you for the tips.

Reply:
CSS can be tricky. :-)